articles/api/v1/use-cases.md

```diff
@@ -11,28 +17,28 @@ Currently, there are certain features and functionality that are only available
 
 ## Active Directory Connector Monitoring
 
-In APIv1, there is a `GET` endpoint that allows you to monitor the status of your Active Directory Connector:
+In API v1, there is a `GET` endpoint that allows you to monitor the status of your Active Directory Connector:
 
 `/api/connections/{AUTH0_CONNECTION}/socket`
 
 ## Client Users
 
-With APIv1, after authenticating with the `client_id` and the `client_secret` of an application, you may make a `GET` call to the [appropriate Users endpoint](https://auth0.com/docs/api/v1#!#get--api-clients--client-id--users) to return only those users that belong to a Connection enabled on that application.
+With API v1, after authenticating with the `client_id` and `client_secret` of an application, you can make a `GET` call to the [appropriate Users endpoint](/api/v1#!#get--api-clients--client-id--users) to return only those users that belong to any specified client connection that is enabled for that application.
 
 ## Email
 
-In APIv1, as part of your automation process, you may use the `PATCH` email endpoint to update email templates.
+With API v1, you can use the `PATCH` email endpoint to update email templates as part of your automation process.
 
```

Comment:
Sorry, this is a PUT:

https://auth0.com/docs/api/v1#put--api-emails--email-template-name-